airflow_home/scripts/extract.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+
+API_URL = "https://yfapi.net/v6/finance/quote"
+
+
+def extract(symbol: str, **kwargs) -> None:
+    ti = kwargs['ti']  # need for xcom push and pull commands
+
+    load_dotenv()
+
+    headers = {
+        'x-api-key': os.environ.get('API_KEY')
+    }
+
+    ticker_data = dict()
+
+    querystring = {"symbols": symbol}
+
+    response = requests.request("GET", API_URL, headers=headers,
+                                params=querystring)
```

Comment:
`requests.request` is a low-level construction. If you don't need custom methods or low-level configuration, you can use `requests.get` instead.